Functions/QualityControl/xASL_qc_GenerateReport.m

```diff
@@ -0,0 +1,418 @@
+function [x,config] = xASL_qc_GenerateReport(x, subject)
+% xASL_qc_GenerateReport Print output QC
+%
+% FORMAT: xASL_qc_GenerateReport(x)
```

Comment:
[, subject]) is missing here

Reply:
also - is there is is there not output?

Reply:
Why did you resolve this conversation? Did you understand his comment (it is about line 4, not line 1)?